### DISCUSSION TITLE
Who built the Colosseum and when?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#977] [Galaxy Trivia] [answered]
**Bonus:** How many arcs does it have?

**Comment by peterbaccaro** [comment, marked as answer]
The Colosseum was built by the **ancient Romans** during the Flavian dynasty, which lasted from 69 to 96 AD. The construction of the Colosseum was initiated by the Emperor Vespasian and completed by his son, Titus.

The Colosseum, was **built between C.E. 70 and 72** and was enjoyed by Roman citizens during the height of the Roman Empire.

The Colosseum has a total of **240 exterior arches**. The first three tiers each contain 80 arches each, while the 4th and top tier contains around 30 small rectangular windows. Each of these arches once held a statue of a prominent Roman citizen, god or goddess.
